ola.spec.in

```diff
@@ -1,10 +1,9 @@
-#missing el6 macros
-%if 0%{?el6}
```

Comment:
no, it just means the shims do their work whenever they need to and not only on el6